c/libeulabeia/include/eulabeia/types.h

```diff
@@ -281,23 +301,93 @@ struct EulabeiaPorts {
 	unsigned int len;
 	unsigned int cap;
 };
+
+struct EulabeiaPluginReference {
+	char *id;
+	char *name;
+	char *value;
+	char *type;
+	char *description;
+	char *defaultvalue;
+};
+
+struct EulabeiaPluginReferences {
+	struct EulabeiaPluginReference *reference;
+	unsigned int len;
+	unsigned int cap;
+};
+
+struct EulabeiaPluginParameter {
+	char *type;
+	char *id;
+};
```

Comment:
Mixed up Reference and Parameter. Their members are reversed. See: https://github.com/greenbone/eulabeia/blob/main/models/target.go#L28